database/src/schema.rs

```diff
@@ -1 +0,0 @@
-infer_schema!("dotenv:DATABASE_URL");
```

Comment:
This removal ( and replacement via manual schema generation ) was due to SQLite throwing several errors using `infer_schema` in the tests; since Diesel seems to be deprecating `infer_schema` anyways, it seemed a positive direction.